clang/include/clang/Basic/Cuda.h

```diff
@@ -40,15 +40,16 @@ enum class CudaVersion {
   CUDA_120,
 #ifdef SYCLomatic_CUSTOMIZATION
   CUDA_121,
-  FULLY_SUPPORTED = CUDA_121,
+  CUDA_122,
+  FULLY_SUPPORTED = CUDA_122,
   PARTIALLY_SUPPORTED =
-      CUDA_121, // Partially supported. Proceed with a warning.
+      CUDA_122, // Partially supported. Proceed with a warning.
```

Comment:
I am curious about the semantics here. Is CUDA 12.2 fully supported or not?

Reply:
No. SYCLomatic only claims to partially migrate the original Code.